x-pack/plugins/infra/public/components/asset_details/constants.ts

```diff
@@ -5,4 +5,5 @@
  * 2.0.
  */
 
+export const ASSET_DETAILS_FLYOUT_COMPONENT_NAME = 'infraAssetDetailsFlyout';
```

Comment:
Do we want to have a different name in case it's open as page? Maybe I am missing something but I don't see a different `component` value or passing `renderMode` to this event 🤔

Reply:
Yes. the name will be different. Probably `infraAssetDetailsPage`